doc/design/audit-trail-mvp.md

```diff
@@ -0,0 +1,157 @@
+
+2. **Display of audit log (to owners and admins only).** The Dandiset landing
+   page will display a rendered (i.e. human-readable) version of the log event
+   data (as long as the viewing user is an owner of the Dandiset, or an admin).
+
+   A possible example of such a display follows (the final rendered form may
+   differ from this depending on best practices, etc.):
+    
+    ```
+    YYYYMMDDTHHMMSS.mmmm: X added asset at path (/full/path) checksum
+    YYYYMMDDTHHMMSS.mmmm: X updated asset at path (/full/path) checksum
+    YYYYMMDDTHHMMSS.mmmm: X removed asset at path (/full/path) checksum
+    YYYYMMDDTHHMMSS.mmmm: X updated dandiset metadata
+    ```
+    
+3. **Extensible significant action list.** The significant actions the system is
+   capable of tracking should be extensible--that is, in the future new actions
+   should be able to be added to the system, and existing actions should be able
+   to expand the specific data they carry.
+
+## Significant Actions
+
+The following is the list of actions performed on Dandisets that are considered
+"significant" (i.e., require an audit record to be generated), along with any
+additional, event-specific data each will carry with it.
+
+| Action                  | General data                    | Specific data                                                                   |
+| ---                     | ---                             | ---                                                                             |
+| Create Dandiset         | date_time, user_id, dandiset_id | metadata, embargoed                                                             |
+| Add owner               | date_time, user_id, dandiset_id | owner_id                                                                        |
+| Remove owner            | date_time, user_id, dandiset_id | owner_id                                                                        |
+| Update Version metadata | date_time, user_id, dandiset_id | metadata                                                                        |
+| Add asset               | date_time, user_id, dandiset_id | asset_path, checksum, metadata, blob_id, asset_id                               |
+| Update asset            | date_time, user_id, dandiset_id | asset_path, old_asset_path, checksum, blob_id, metadata, asset_id, old_asset_id |
+| Remove asset            | date_time, user_id, dandiset_id | asset_path, asset_id                                                            |
+| Unembargo Dandiset      | date_time, user_id, dandiset_id | (none)                                                                          |
+| Publish Dandiset        | date_time, user_id, dandiset_id | version                                                                         |
+| Delete Dandiset         | date_time, user_id, dandiset_id | (none)                                                                          |
+
```

Comment:
```suggestion

Note that we explicitly are not aiming in this MVP to audit individual Blob and Zarr uploads.

```

if that is the case... 

But my question is: from the DB or other logs -- would we be able to figure out which user_id has uploaded a blob with porn which we are now happily serving through our `/blobs/digest` endpoint?